src/clj/rems/auth/fake_login.clj

```diff
@@ -42,25 +51,38 @@
 
 (defn- user-selection [username]
   (let [url (url (login-url) {:username username})]
-    [:div.user.mb-3.mr-3.flex-fill.d-flex {:onclick (str "window.location.href='" url "';")}
-     [:a.btn.btn-primary.flex-fill {:href url} username]]))
+    [:div.user.m-3 {:onclick (str "window.location.href='" url "';")}
+     [:a.btn.btn-primary {:href url :style "width: 12rem; overflow: hidden; text-overflow: ellipsis; white-space: nowrap"}
+      username]]))
 
 (defn- fake-login-screen [{session :session :as req}]
   (if-let [username (-> req :params :username)]
     (fake-login session username)
     (layout/render nil
-                   {:app-content [:div.d-flex.justify-content-start.align-items-center
+                   {:app-content [:div.container
                                   [:div.row.w-100
                                    [:div.logo.w-100
                                     [:div.container.img]]]
-                                  [:div.login.row.d-flex.justify-content-center.align-items-center
-                                   [:div.col-md-8
-                                    [:h1.text-center "Development Login"]
-                                    [:div.users.d-flex.flex-wrap.justify-content-stretch.align-items-start
-                                     (->> (get-fake-users)
-                                          sort
-                                          distinct
-                                          (map user-selection))]]]]})))
+                                  [:div.login.row.d-flex.flex-column.justify-content-start.align-items-center.mb-5
+                                   [:h1.text-center "Development Login"]
+                                   [:div.users
+                                    (for [{:keys [group users]} (get-fake-user-descriptions)
+                                          :let [id (escape-element-id group)]]
+                                      [:div
+                                       [:h2.mx-3 group]
+
+                                       (when (= "Special Users" group)
+                                         [:div.ml-3
+                                          [:p "Special users with special cases. Not for daily use!"]
+                                          [:button#show-special-users.btn.btn-secondary {:data-toggle "collapse" :data-target (str "#" id)}
+                                           "Show"]])
```

Comment:
perhaps this could be configured in the group data as well?
```clojure
{:group "Special Users"
 :description "Special users with special cases. Not for daily use!"
 :collapsed true
 :users [...]}
```

Reply:
Since there is only one exception with the collapse, I find that it would just add a bunch of generic code that isn't that useful, that actually complicates the code for the screen and the data. Let's do it if we find another use for it.

Reply:
Added a comment about this possible refactoring.